### PR TITLE
Rebrand from Piwik to Matomo

### PR DESCRIPTION
**Pull request by sgiehl**
Needs to be republished on packagist after merge with the new name, and the old one needs to be marked as `Abandoned`.

### CODE REVIEW

File: MatomoTracker.php [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+ * MatomoTracker implements the Matomo Tracking Web API.
  *
  * For more information, see: https://github.com/matomo-org/matomo-php-tracker/
  *
- * @package PiwikTracker
+ * @package MatomoTracker
  * @api
  */
-class PiwikTracker
+class MatomoTracker
```
Comment (tsteur) [inline MatomoTracker.php]: Can you add a `class PiwikTracker extends MatomoTracker` or so for BC?

Comment (sgiehl) [reply MatomoTracker.php]: Sure can do that. But actually I thought we will break BC nevertheless with Matomo 4

Comment (tsteur) [reply MatomoTracker.php]: I guess if we can keep BC so easily, there be no need for it maybe.

Comment (sgiehl) [reply MatomoTracker.php]: done

**Comment by tsteur** [comment]
Just a random thought... wondering if it makes sense to have `PiwikTracker.php` file as well including `MatomoTracker.php` and then in both PiwikTracker and MatomoTracker class we could have

`if (!class_exists(PiwikTRacker)) { . class PiwikTracker extends MatomoTracker }`? This way BC works no matter if Piwiktracker file was included or only Matomotracker? Just makes things bit easier even for us as we use PiwikTracker in various places and be great if we give people an easy way to update... otherwise everyone needs to check which file exists and while file should be loaded etc

In Matomo 5 we could then remove PiwikTracker completely (or leave it there if it doesn't cause us any issues)

**Comment by sgiehl** [comment]
@tsteur updated the PR

**Comment by tsteur** [comment]
👍

**Pull request merged**